staging/src/k8s.io/apiserver/pkg/server/options/serving.go

```diff
@@ -286,3 +285,22 @@ func (s *SecureServingOptions) MaybeDefaultWithSelfSignedCerts(publicAddress str
 
 	return nil
 }
+
+func (s *SecureServingOptions) CreateListener() error {
```

Comment:
Better make this functional, i.e. without mutation and the listener as return value.

Reply:
ok